### DISCUSSION TITLE
The Weather Network [CA]

### DISCUSSION THREAD

**Discussion by Metroid2023** [iptv-org/community#925] [Q&A]
### Channel ID

TheWeatherNetwork.ca

### Notes (optional)

Cable and Satellite feed.

SD
HD

### Please confirm the following

- [x] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by clips519** [comment]
here ya go weather network canada

https://d3f6rv2ihfj09x.cloudfront.net/v1/master/3722c60a815c199d9c0ef36c5b73da68a62b09d1/cc-4l4ssesb90374-ssai-prd/playlist.m3u8?ads.device_did=%7BPSID%7D&ads.device_dnt=%7BTARGETOPT%7D&ads.app_domain=%7BAPP_DOMAIN%7D&ads.app_name=%7BAPP_NAME%7D

**Comment by Metroid2023** [comment]
Thank you, but this is The Weather Network FAST Channel. Does anyone have The Weather Network from cable? I found them here at :https://d.tousecurity.com/home18/ ,but it keeps on freezing and stop working.

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/ca_pluto.m3u#L390-L391
